packages/ajv-validator/tsconfig.build.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist",
+    "tsBuildInfoFile": "dist/tsconfig.build.tsbuildinfo",
+    "emitDeclarationOnly": false
```

Comment:
We can change this as we use `tsup` to build the output.

One benefit of having `tcs` emit declaration files during the build is it can generate [declaration maps](https://www.typescriptlang.org/tsconfig/#declarationMap). They enable features like "Go to Definition" in tools like vscode. There are some conversations as to if these files are [worth publishing to npm](https://github.com/egoist/tsup/issues/885), `tsup` also [mentions it in their documentation](https://github.com/egoist/tsup/blob/main/docs/README.md#generate-typescript-declaration-maps-dtsmap).

There are also a few issues about this:
- https://github.com/egoist/tsup/issues/571
- https://github.com/egoist/tsup/issues/647